scripts/audio.js

```diff
@@ -317,3 +330,42 @@ H5P.Audio.prototype.getCurrentState = function () {
     };
   }
 };
+
+/**
+ * @public
+ * Disable button.
+ * Not using disabled attribute to block button activation, because it will
+ * implicitly set tabindex = -1 and confuse ChromeVox navigation. Clicks handled
+ * using "pointer-events: none" in CSS, keydown on enter or space prevented by
+ * interception in event listener on button.
+ */
+H5P.Audio.prototype.disable = function () {
+  if (this.$audioButton) {
+    this.$audioButton.toggleClass(H5P.Audio.BUTTON_DISABLED, true);
+  }
+};
```

Comment:
I think disable/enable could be misunderstood. I propose renaming to disableToggleButton()/enableToggleBurtton() or something better :)

Reply:
changed, also changes isEnabled to isEnabledToggleButton to prevent future confusion